geniza/corpus/admin.py

```diff
@@ -167,14 +167,15 @@ class DocumentAdmin(admin.ModelAdmin):
         "has_image",
         "is_public",
     )
-    readonly_fields = ("created", "last_modified", "shelfmark", "id")
+    readonly_fields = ("created", "last_modified", "shelfmark", "id", "old_pgpids")
     search_fields = (
         "fragments__shelfmark",
         "tags__name",
         "description",
         "notes",
         "needs_review",
         "id",
+        "old_pgpids",
```

Comment:
ooh, I should have added a comment here — these search fields are ignored since we use solr for document search. We should add old pgpids to the index data and admin search, though. If that's not obvious to you I can throw it in!

Reply:
Not sure if I'm jumping into the wrong conversation, but saw on Slack the earlier comment... @kmcelwee - I think we do need to be able to search for old PGP IDs! Especially with all of the demerge work happening, it's crucial we have access to searching this way.